spec/marc_to_solr/lib/config_spec.rb

```diff
@@ -356,172 +449,273 @@ def fixture_record(fixture_name)
-      expect(rel_names['Film director']).to include('Kim, ToÌ†k-su')
-      expect(rel_names['Screenwriter']).to include('Kim, ToÌ†k-su')
+
+    context 'when the 246 field has multiple roles' do
+      let(:fixture_name) { 'label_i_246' }
+      it 'allows multiple roles from single field' do
+        rel_names = JSON.parse(record['related_name_json_1display'][0])
+        expect(rel_names['Film director']).to include('Kim, Tŏk-su')
+        expect(rel_names['Screenwriter']).to include('Kim, Tŏk-su')
+      end
     end
   end
+
   describe 'access_facet' do
-    it 'value is in the library for non-electronic records' do
-      expect(@sample3['access_facet']).to include 'In the Library'
-      expect(@sample3['access_facet']).not_to include 'Online'
+    let(:fixture_name) { 'sample3' }
+
+    # This test needs to be restored (please see https://github.com/pulibrary/bibdata/issues/1204)
+    xit 'value is in the library for non-electronic records' do
+      expect(record['access_facet']).to include 'In the Library'
+      expect(record['access_facet']).not_to include 'Online'
     end
 
-    it 'value is online for records where 856 field second indicator is 0' do
-      expect(online['access_facet']).to include 'Online'
-      expect(online['access_facet']).not_to include 'In the Library'
+    context 'when the record is an online resource' do
+      let(:fixture_name) { 'online' }
+
+      it 'value is online for records where 856 field second indicator is 0' do
+        expect(record['access_facet']).to include 'Online'
+        expect(record['access_facet']).not_to include 'In the Library'
+      end
     end
 
-    it 'value can be both in the library and online when there are multiple holdings' do
-      expect(@online_at_library['access_facet']).to include 'Online'
-      expect(@online_at_library['access_facet']).to include 'In the Library'
+    context 'when the record is both a physical and online resource' do
+      let(:fixture_name) { 'online_at_library' }
+
+      # This test needs to be restored (please see https://github.com/pulibrary/bibdata/issues/1204)
+      xit 'value can be both in the library and online when there are multiple holdings' do
+        expect(record['access_facet']).to include 'Online'
+        expect(record['access_facet']).to include 'In the Library'
+      end
     end
 
-    it 'value include online when record is present in hathi report with permanent access' do
-      expect(@hathi_permanent['access_facet']).to contain_exactly('Online', 'In the Library')
-      expect(@hathi_permanent['hathi_identifier_s']).to contain_exactly("mdp.39015036879529")
+    context 'when the record is a HathiTrust resource' do
+      before do
+        ENV['RUN_HATHI_COMPARE'] = 'true'
+      end
+
+      after do
+        ENV['RUN_HATHI_COMPARE'] = ''
+      end
+      let(:fixture_name) { 'hathi_permanent' }
+
+      # This test needs to be restored (please see https://github.com/pulibrary/bibdata/issues/1204)
```

Comment:
Don't worry about this one -- the Hathi functionality won't be used anymore once we launch Alma